### ISSUE TITLE
Manually add status codes for retry

### ISSUE THREAD

**Issue by awwwer** [rodion-m/dio_smart_retry#11]
Would it be possible to add specific status codes beside the default codes to 'dio_smart_retry'?
For example I would like to retry on status code 401, but it is not included. 
Please add an option to set optional status codes. Thank you!

**Comment by rodion-m** [comment]
Hi @awwwer ! Thank you for contributing. \
Di you find this API useful?
```dart
    RetryInterceptor(
      dio: dio,
      logPrint: print,
      retries: 3,
      retryableExtraStatuses: { status401Unauthorized }, //<--- proposal way to set extra retriable statuses
    ),
```

**Comment by awwwer** [comment]
Hi @rodion-m ,
yes I think so! 
But this is not implemented yet, right?

**Comment by rodion-m** [comment]
Feature was implemented in v1.3 (https://github.com/rodion-m/dio_smart_retry/commit/22dd6705787d78273a09bec77cfdbf4aa13eb669)
